ethcore/src/state/account.rs

```diff
@@ -315,6 +315,11 @@ impl Account {
 		self.code_hash == SHA3_EMPTY
 	}
 
+	/// Check if account is basic (Has no code).
+	pub fn is_basic(&self) -> bool {
+		self.code_hash == SHA3_EMPTY
```

Comment:
If it's  newly created account that does not have code it is a basic account, right? It stops being basic only after it gets the code.